Product/Production/Services/PatientDiscoveryCore/src/main/java/gov/hhs/fha/nhinc/patientdiscovery/PatientDiscovery201305Processor.java

```diff
@@ -125,9 +128,9 @@ protected boolean checkPolicy(PRPAIN201306UV02 response, AssertionType assertion
         PatientDiscovery201306PolicyChecker policyChecker = PatientDiscovery201306PolicyChecker.getInstance();
 
         // ************************************************************************************************
+        boolean hasEmptySubject = checkEmptySubject(response);
```

Comment:
Are you sure that using the checkEmptySubject() returns the same expected boolean value with what was in line 129 originally?